1_fetch/src/get_nwis_data.R

```diff
@@ -1,15 +1,18 @@
 
-download_nwis_data <- function(site_nums = c("01427207", "01432160", "01435000", "01436690", "01466500")){
-  
-  # create the file names that are needed for download_nwis_site_data
-  # tempdir() creates a temporary directory that is wiped out when you start a new R session; 
-  # replace tempdir() with "1_fetch/out" or another desired folder if you want to retain the download
-  download_files <- file.path(tempdir(), paste0('nwis_', site_nums, '_data.csv'))
+make_table <- function(site_data_dir){
+  file_names = dir(site_data_dir)
+  m5sums = as.vector(md5sum(site_data_dir))
+  df = data.frame('filename' = file_names, 'm5sums'=m5sums)
```

Comment:
Nice! There's a pre-built version of this function called `combine_to_ind` (or `sc_indicate` is very similar, just a different argument structure) in scipiper. With either of those, if you set `ind_file=''`, you'll get an R object back instead of getting a file written.